### PR TITLE
Create User Type Component for Sign Up page.

### LINKED ISSUE #15: Sign Up - User Type Component
Implement the user type component based on the Figma design,

It should have a select and the user types are received from the backend.  

`${apiUrl}/get-donor-types/`

### PR DESCRIPTION
**Pull request by dianapaula19**
### What does it fix?

Closes #15

* create redux action for setting the user type
* create user type component
* add an extra option for the dropdown component (which is hidden and disabled)

### CODE REVIEW

File: store/reducers/steps/index.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const defaultStepsState: StepsStore = {
   steps: [
     {
       label: 'steps.userType',
-      component: 'TipUtilizator',
+      component: <UserTypeForm />
```
Comment (cipick) [inline store/reducers/steps/index.tsx:15]: we should not keep the component in the redux store as we need to be keep the store serializable.
i recommend keeping a string key here, and in the component where the `steps` value is used to [dynamic import](https://nextjs.org/docs/advanced-features/dynamic-import) the `UserTypeForm` if `step.component === "TipUtilizator"`

**Review by cipick — CHANGES_REQUESTED** [review]

File: store/reducers/steps/index.tsx (hunk 1 above)
Comment (cipick) [reply store/reducers/steps/index.tsx:15]: Screenshot 2022-03-01 at 16 55 04
this should help with performance too

Comment (dianapaula19) [reply store/reducers/steps/index.tsx]: Done :heavy_check_mark:

**Review by cipick — APPROVED** [review]
Awesome 👍

**Pull request merged**